cnf-certification-test/performance/suite.go

```diff
@@ -79,6 +85,22 @@ var _ = ginkgo.Describe(common.PerformanceTestKey, func() {
 	// Scheduling related tests ends here
 })
 
+func testLimitedUseOfExecProbes(env *provider.TestEnvironment) {
+	var compliantObjects []*testhelper.ReportObject
+	var nonCompliantObjects []*testhelper.ReportObject
+	for _, put := range env.Pods {
+		for _, cut := range put.Containers {
+			if cut.LivenessProbe.PeriodSeconds < 10 && cut.LivenessProbe.InitialDelaySeconds <= 10 {
+				nonCompliantObjects = append(nonCompliantObjects, testhelper.NewPodReportObject(put.Namespace, put.Name, "Ensure limited use of exec probes", false))
```

Comment:
The report object should match the object being tested. In this case, you should report containers. Adding the content of the exec probe would also be helpful.